script.js

```diff
@@ -0,0 +1,93 @@
+// the book object
+const book = {
+  title: '',
+  author: '',
+};
+
+// Check local storage for books collection object
+let booksCollection = JSON.parse(localStorage.getItem('books')) || [];
+
+// Save books to local storage
+function saveBooks(books) {
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+// Remove an existing book function
```

Comment:
- [ ] [OPTIONAL] I actually think the comments are too much, and probably aren't needed, if I should see a `removeBook` function in a code, I most likely will know what that does 😄.